ui/app/models/node.js

```diff
@@ -64,4 +64,14 @@ export default Model.extend({
   compositeStatus: computed('status', 'isEligible', function() {
     return this.isEligible ? this.status : 'ineligible';
   }),
+
+  state: computed('isDraining', 'isEligible', 'status', function() {
+    if (this.isDraining) {
+      return 'draining';
+    } else if (!this.isEligible) {
+      return 'ineligible';
+    } else {
+      return this.status;
+    }
+  }),
```

Comment:
Hmmmm. I think we can drop `compositeStatus` in favor of `state` (but maybe keep the compositeStatus name? idk) with little effort or risk. We could add a css class for the node status light for draining that is equivalent to ineligible.

Reply:
Agreed, I did this in ad3863f. One caveat is that I removed an assertion in a client detail test:

```
    assert.equal(
      ClientDetail.statusLight.objectAt(0).id,
      node.status,
      'Title includes status light'
    );
```

Being able to test that with randomness was unreliable and would require the duplication of the `compositeStatus` computed property in the test. Since there’s another test that specifically sets a node to `ineligible` and checks for that, it seemed to me safe to remove.

I did make `.draining` equivalent to `.ineligible` for the light, but I wonder whether it wouldn’t make sense to use the blue draining colour that shows in the client details boxed section and the state column of the client list? For consistency.

Also, I’ve been surprised that hovering over the status light doesn’t have any description of what the colour means; it’s true that it can usually be decoded by looking for the corresponding colour in the client details boxed section, but maybe a tooltip would be a good idea to be more explicit?

Reply:
Good thoughts on ways to improve that indicator. I know @joshklekamp noodled with it too as part of the drain work (added icons, thought through colors). 

I forget if he added a draining state. Either way, worth discussing further but out of scope for this bug fix.